### ISSUE TITLE
False negative for Style/MultipleComparison when not part of conditional

### ISSUE THREAD

**Issue by asherkach** [rubocop-hq/rubocop#5467]
The `Style/MultipleComparison` cop fails to trigger if the multiple comparison is not part of a conditional.

--------

## Expected behavior

A Style/MultipleComparison violation is triggered for
```
def foo(x)
  x == 1 || x == 2 || x == 3
end
```
as this could be written more succinctly as
```
def foo(x)
  [1, 2, 3].include? x
end
```

## Actual behavior

No Style/MultipleComparison violation is reported.

## Steps to reproduce the problem

$ cat bug.rb 
def foo(x)
  x == 1 || x == 2 || x == 3
end
$ bundle exec rubocop --only Style/MultipleComparison bug.rb
Inspecting 1 file
.

1 file inspected, no offenses detected

## RuboCop version

```
$ rubocop -V
0.52.1 (using Parser 2.4.0.2, running on ruby 2.2.3 x86_64-darwin16)
```
